main/src/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -1952,4 +1952,27 @@ public static String getUsername(final String page) {
         return StringUtils.isNotEmpty(usernameOld) ? usernameOld : null;
     }
 
+    @Nullable
+    public static int getCachesCount(final String page) {
+        int cachesCount = -1;
```

Comment:
really cachesCount=-1 if it could not determined?
@Lineflyer What do you mean from support point of view?

Reply:
Somewhere it was mentioned, that -1 is not determined, where as 0 means no finds

Reply:
In MainActivity.fillView the FoundNumCounter is only updated, if getCachesFound>= 0, so no -1 shown to the user. FoundNumCounter stays then unchanged